skeleton/release.nix

```diff
@@ -0,0 +1,46 @@
+{ supportedSystems ? [ "x86_64-linux" "x86_64-darwin" ]
+, supportedCrossSystems ? [ "x86_64-linux" ]
+, scrubJobs ? true
+, iohk-skeleton ? { outPath = ./.; rev = "abcdef"; }
+, projectArgs ? { config = { allowUnfree = false; inHydra = true; }; }
+, iohkLib ? import ./nix/iohk-common.nix {}
+}:
+
+with (import iohkLib.release-lib) {
```

Comment:
Here I'm getting the [following error](https://buildkite.com/input-output-hk/decentralized-software-updates/builds/4#7d1d77c3-f2ee-414a-91e2-0be0df56419a):

```

warning: `--gc-roots-dir' not specified
--
  | {
  | "": {
  | "error": "attribute 'release-lib' missing, at /var/lib/buildkite-agent/builds/buildkite-packet-1/input-output-hk/decentralized-software-updates/release.nix:9:14"
  | }
  | }
  | ERROR: evaluation completed in 0.01 seconds with 1 errors
  | : attribute 'release-lib' missing, at /var/lib/buildkite-agent/builds/buildkite-packet-1/input-output-hk/decentralized-software-updates/release.nix:9:14

```

Reply:
@dnadales Ah - that attribute is actually new in this PR, so you will need to set your iohk-nix pin to this branch.

Reply:
Oh, ok, but the build also fails in this PR as well :)

Reply:
Sorry about that. The reason is because it was importing iohk-nix from the git master branch, which doesn't yet have that attribute.

Reply:
:pray:  so, in my the project I want to add CI support, do I wait till this PR is merged?